common/lib/capa/capa/capa_problem.py

```diff
@@ -732,6 +734,7 @@ def _extract_html(self, problemtree):  # private
                 'status': status,
                 'id': input_id,
                 'input_state': self.input_state[input_id],
+                'question_label': self.question_label if self.question_label else None,
```

Comment:
If `self.question_label == ""`, the `if` will fail and the `None` will be put into the dictionary. Is this the expected behaviour?